src/ripple/basics/hardened_hash.h

```diff
@@ -59,12 +59,12 @@ make_seed_pair() noexcept
     {
         std::mutex mutex;
         std::random_device rng;
-        std::mt19937_64 gen {rng()};
+        std::mt19937_64 gen;
         std::uniform_int_distribution <std::uint64_t> dist;
 
         state_t() : gen(rng()) {}
-        // state_t(state_t const&) = delete;
-        // state_t& operator=(state_t const&) = delete;
+        state_t(state_t const&) = delete;
+        state_t& operator=(state_t const&) = delete;
```

Comment:
I'm sure that's the reason too.  But there are now 278 occurrences of "= delete" in rippled.  I think what probably happened is that I mistakenly thought VS-2013 didn't support "= delete" and I was wrong.